### PR TITLE
order payment added

### PR DESCRIPTION
**Pull request by Evgeny-Gromyko**

### CODE REVIEW

File: Services/Store/ModsenOnlineStore.Store.API/Controllers/OrdersController.cs [hunk 1]
```diff
@@ -39,13 +45,38 @@ public async Task<IActionResult> GetSingleOrderAsync(int id)
...
+            {
+                client.BaseAddress = new Uri("https://localhost:7107/");
+                var response = await client.PostAsJsonAsync("EmaiLoginlAuthentication", "egrom2002@gmail.com");
+                if (response.IsSuccessStatusCode)
+                {
+                    var data = await response.Content.ReadAsStringAsync();
+
+                    order.PaymentConfirmationCode = data;
+                }
+            }
```
Comment (elv1sabene) [inline Services/Store/ModsenOnlineStore.Store.API/Controllers/OrdersController.cs:64]: do you need this code in controller?

File: Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs [hunk 2]
```diff
@@ -69,6 +69,24 @@ public async Task<IActionResult> RegisterUserAsync(AddUserDto user)
             
             return Ok(response.Message);
         }
+        //    return Ok(await service.RegisterUser(user));
+        //}
```
Comment (elv1sabene) [inline Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs]: delete comments

File: Services/Store/ModsenOnlineStore.Store.API/Controllers/OrdersController.cs (hunk 1 above)
Comment (Evgeny-Gromyko) [reply Services/Store/ModsenOnlineStore.Store.API/Controllers/OrdersController.cs:64]: removed

File: Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs (hunk 2 above)
Comment (Evgeny-Gromyko) [reply Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs]: deleted

File: Services/TokenAuthentication/ModsenOnlineStore.Login.Application/Services/LoginService.cs [hunk 3]
```diff
@@ -74,6 +74,25 @@ public async Task<DataResponseInfo<User>> GetUserByIdAsync(int id)
             return new DataResponseInfo<User>(data: user, success: true, message: $"user with id {user.Id}");
         }
 
+        public async Task<ResponseInfo> MakePaymentAsync(int id, decimal money)
```
Comment (elv1sabene) [inline Services/TokenAuthentication/ModsenOnlineStore.Login.Application/Services/LoginService.cs]: I think this method is not responsibility of login service

File: Services/TokenAuthentication/ModsenOnlineStore.Login.Domain/Entities/User.cs [hunk 4]
```diff
@@ -1,3 +1,12 @@
+﻿using Microsoft.EntityFrameworkCore.Metadata.Internal;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (elv1sabene) [inline Services/TokenAuthentication/ModsenOnlineStore.Login.Domain/Entities/User.cs]: delete unnecessary usings

File: Services/TokenAuthentication/ModsenOnlineStore.Login.Application/Services/LoginService.cs (hunk 3 above)
Comment (Evgeny-Gromyko) [reply Services/TokenAuthentication/ModsenOnlineStore.Login.Application/Services/LoginService.cs]: Should I create a separate payment service (user microservice)? Is it worth moving the functions of getting information about the user into a new user microservice?

Comment (elv1sabene) [reply Services/TokenAuthentication/ModsenOnlineStore.Login.Application/Services/LoginService.cs]: Creating of another service (e.g. IPaymentService) will be enough within current microservice

File: Services/TokenAuthentication/ModsenOnlineStore.Login.Domain/Entities/User.cs (hunk 4 above)
Comment (Evgeny-Gromyko) [reply Services/TokenAuthentication/ModsenOnlineStore.Login.Domain/Entities/User.cs]: removed

File: Services/TokenAuthentication/ModsenOnlineStore.Login.Application/Services/LoginService.cs (hunk 3 above)
Comment (Evgeny-Gromyko) [reply Services/TokenAuthentication/ModsenOnlineStore.Login.Application/Services/LoginService.cs]: done

File: Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs [hunk 5]
```diff
@@ -69,13 +71,44 @@ public async Task<IActionResult> RegisterUserAsync(AddUserDTO user)
             return Ok(response.Message);
         }
 
+        [HttpPost]
+        [Authorize(Roles = "User")]
+        [Route("/NewPayment/{userId}")]
```
Comment (elv1sabene) [inline Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs]: you can specify route within `[HttpPost]` attribute

File: Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs [hunk 6]
```diff
@@ -69,13 +71,44 @@ public async Task<IActionResult> RegisterUserAsync(AddUserDTO user)
...
+            {
+                return NotFound(response);
+            }
+
+            return Ok(response);
+        }
+
+        [HttpPost]
+        [Authorize(Roles = "Admin")]
+        [Route("/AddMoney/{userId}")]
```
Comment (elv1sabene) [inline Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs]: the same

**Review by elv1sabene — APPROVED** [review]

File: Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs (hunk 6 above)
Comment (Evgeny-Gromyko) [reply Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs]: endpoint route moved to [HttpPost] attribute

File: Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs (hunk 5 above)
Comment (Evgeny-Gromyko) [reply Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs]: endpoint route moved to [HttpPost] attribute

**Pull request merged**